tests/browser_context_test.go

```diff
@@ -189,3 +192,86 @@ func TestBrowserContextAddCookies(t *testing.T) {
 		})
 	}
 }
+
+func TestBrowserContextCookies(t *testing.T) {
+	t.Parallel()
+
+	// an empty page is required to set cookies. we're just using a
```

Comment:
Thanks for all these comments 🙂 

I feel that the code is self explanatory, and the comments just get in the way. The names of the table tests are perfect, and as more tests are added it helps build the picture of how this test works.

Reply:
The comments are a bit verbose, I agree 👍 They can also help other contributors. Let me know if you want me to remove some of them. Thanks!